### ISSUE TITLE
Preserve numbering style of ordered lists

### ISSUE THREAD

**Issue by mk-pmb** [yzhang-gh/vscode-markdown#544]
Thanks for your great addon!

I found the option to select how to number the second item by default (#283).
However, please preserve the existing style if it is unambiguous:

When adding to a list that has multiple items, …
* and all of them are numbered "1", "a" etc., use all-one style.
* starts at "1" or "a" and all other items are counting continuously, use counting style.

**Comment by yzhang-gh** [comment]
Thanks for the suggestion. It sounds like a good enhancement, although it may be a bit tricky to implement (e.g. nested lists).

Before this is implemented, you can use the `orderedList.marker` per workspace.

**Comment by blakegearin** [comment]
I prefer using all 1s for auto-numbering so ended up turning `markdown.extension.orderedList.autoRenumber` to `false`

**Comment by mk-pmb** [comment]
Yeah that's the easy way if all the projects you work on use the same style, or you're entitled to change their style. I still hope someone will implement this soon, because while I agree that all-one is the best choice, I still participate in lots of projects with different styles.
